src/types/common.d.ts

```diff
@@ -5,11 +5,16 @@ export interface FontInfo {
   style?: string;
 }
 
+export interface Point {
```

Comment:
I'm in two minds about this -- it doesn't feel useful enough for all the indirections. Better to remove this interface and inline the structure explicitly: e.g., `function drawLine(a: {x: number, y: number}, b: {x:number, y:number})`.

Reply:
Sounds good to me!

My default inclination is to remove duplication and give names to things that represent particular concepts (e.g., Points, Vectors, Rects).

I've seen `{ x: number; y: number }` used in a few places in VexFlow and thought it might be good to consolidate them under an interface. (Looks like the TypeScript compiler doesn't generate any extra code; it's just for type checking.)

In this case since it's just {x, y} tuples, I'm happy to inline them everywhere we use them.